### ISSUE TITLE
replies to replies not sent from my blog to the fediverse

### ISSUE THREAD

**Issue by YourAutisticLife** [Automattic/wordpress-activitypub#401]
So I produce an article on my blog, and it does to the fediverse.

Someone replies to my article on the fediverse, and the reply also goes on my blog. Yay!

Now, when I reply to this person on my blog, I see the reply on my blog, but nothing is sent to the fediverse. Bummer!

I'd don't see my last reply and neither the person I replied to does. (Yes, I asked. I wanted to make sure nothing unexpected was happening.)

Version of the plugin 0.17.0
Version of WP 6.3

I can add more if needed.

**Comment by pfefferle** [comment]
Sorry, but this is not implemented yet!

**Comment by YourAutisticLife** [comment]
Yikes... and I guess there's no way to prevent people from replying to the post that my blog puts out every time I publish a new article.

**Comment by mediaformat** [comment]
@YourAutisticLife There isn't any agreed ActivityPub method yet to prevent or control who can [reply to a post](https://socialhub.activitypub.rocks/t/fep-5624-per-object-reply-control-policies/2723). 

The choice to include those replies and display them as comments on your site is yours though.

**Comment by msmith9835** [comment]
Is there any time frame on when this feature might be added?

**Comment by pfefferle** [comment]
We are actually working on it https://github.com/Automattic/wordpress-activitypub/pull/142 :)

**Comment by pfefferle** [comment]
It's one of the focus topics for the next iteration!

**Comment by msmith9835** [comment]
Awesome!  Thank you!

**Comment by janboddez** [comment]
Is there a way to reply to replies, from your blog? Like, if comments don't federate ... can we post a new post and mention the other (Mastodon) account and maybe even link to the status (maybe using the `(u-)in-reply-to` microformat or something, just thinking out loud) and have the person receive a notification?

**Comment by pfefferle** [comment]
give us some more days and it will work out of the box!

**Comment by pfefferle** [comment]
You can try the latest version here on GitHub!

**Comment by YourAutisticLife** [comment]
It does work in the latest version.... great!!!

However, the behavior is confusing. I replied to a post of mine. I have two accounts: one tied to my WordPress instance, and one tied to a Mastodon instance of mine. The scenario is as follows:

1. An article is posted to the fediverse by my WordPress instance.
2. I commented on it with my Mastodon account.
3. Then I went into the comment page of my article (in number 1) and replied to the reply in number 2.

I saw the reply but I was confused that I did NOT get a notification for it in Mastodon. I looked at the post closer and I noticed that the reply does not automatically @-mentions the person that you are replying to. 

If I add the @-mention manually, then I get a notification.

Is it possible to pre-populate the reply field with the handle of the person I'm replying to (and actually all the other user handles used in that post, like Mastodon does)?
